src/astro/utils/postgres_transform.py

```diff
@@ -15,7 +15,10 @@
 """
 import inspect
 
-from airflow.providers.postgres.hooks.postgres import PostgresHook
+try:
+    from airflow.providers.postgres.hooks.postgres import PostgresHook
+except ModuleNotFoundError:
+    from astro.utils.schema_util import RaiseException as PostgresHook
```

Comment:
I think you need something like this instead?

```suggestion
try:
    from airflow.providers.postgres.hooks.postgres import PostgresHook
except ModuleNotFoundError:
    from astro.utils.schema_util import RaiseException

    PostgresHook = RaiseException("airflow.providers.postgres.hooks.postgres")
```

Also I’d change the name from `RaiseException`; at first glance I thought the class is subclassing `Exception` (it’s not).

Reply:
Thanks TP, I have made the changes you suggested.